src/component/context/ContextList/ContextList.tsx

```diff
@@ -79,6 +82,7 @@ const ContextList = () => {
                     show={
                         <Tooltip title="Edit context field">
                             <IconButton
+                                aria-label="edit"
```

Comment:
`Tooltip` will add a `title` attribute to the button, so I don't think we need these.